manifests/base/commit-latest.env

```diff
@@ -1,20 +1,20 @@
-odh-pipeline-runtime-datascience-cpu-py312-ubi9-commit-n=d91bb6a
```

Comment:
this may be well, good, and true, but, why update it in a PR that has nothing to do with this?